app/controllers/api/v1/messages_controller.rb

```diff
@@ -2,19 +2,21 @@ module Api
   module V1
     class MessagesController < Api::V1::ApiController
       def index
-        @messages = policy_scope(conversation.messages).includes(:user)
+        conversation = find_conversation
+        @messages = policy_scope(conversation.messages)
+        render_messages(@messages)
```

Comment:
why do you need this `render_messages` method if you have de create view defined ? I think the previous version was ok

Reply:
Yep in both cases i misunderstood, i think now its ok.

Reply:
Sorry i didnt push the change yet.